config/crd/bases/devops.kubesphere.io_pipelines.yaml

```diff
@@ -4,7 +4,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.6.2
+    controller-gen.kubebuilder.io/version: v0.4.0
```

Comment:
I prefer to use the old version if you don't have a good enough reason to change it.